### PR TITLE
docs(fix): remove react specific code from angular/vue theme page

### PR DESCRIPTION
**Pull request by hbuchel**
#### Description of changes

This PR re-orders some content on the /theming/default-theme page and removes React specific code that was showing up for Angular/Vue.

#### Issue #, if available

N/A
#### Description of how you validated changes

Local docs instance

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zchenwei — APPROVED** [review]
LGTM

**Review by joebuono — APPROVED** [review]
LGTM. Just one question

**Review by dbanksdesign — APPROVED** [review]
LGTM!

File: docs/src/pages/[platform]/theming/default-theme/tokenUsage.react.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Fragment } from '@/components/Fragment';
+
+This structure allows us to reference the small space value as `tokens.space.small` such as in the following example. You can also learn more about [theming](/theming#getting-started) in our getting started guide.
+
+```jsx
+import { useTheme } from '@aws-amplify/ui-react';
+
+const MyComponent = () => {
+  const { tokens } = useTheme();
+  return <View marginBottom={tokens.space.small}>{children}</Text>;
```
[the remark this answers is not in the document]
Comment (hbuchel) [reply docs/src/pages/[platform]/theming/default-theme/tokenUsage.react.mdx:10]: Yeah, also just realized  I have <View></Text> 😅

**Review by zchenwei — APPROVED** [review]
LGTM!

**Pull request merged**